shared/admin-tools/util.ts

```diff
@@ -4,10 +4,12 @@ import { DynamoDBDocument } from '@aws-sdk/lib-dynamodb';
 import readline from 'readline';
 
 const { ENV } = process.env;
-const UserPoolCache = {};
+const UserPoolCache: Record<string, string> = {};
 
-// Look up the current version so that we can perform searches against it
-export const getVersion = async (): Promise<string> => {
```

Comment:
This method is also used in the `become-user` script (as well as a pair of one-off scripts), which I don't see updated in this PR.

Rather than breaking scripts that use it, perhaps we could still export a `getVersion` method that calls this method and returns the `version` string?